
class TournamentError(Exception):
    pass

class TeamError(Exception):
    pass

